bitnami/fluentd/templates/forwarder-configmap.yaml

```diff
@@ -101,10 +105,12 @@ data:
         flush_interval 5s
       </buffer>
     </match>
-    {{- else }}
+    {{ else }}
     # Send the logs to the standard output
     <match **>
       @type stdout
     </match>
```

Comment:
In the aggregator, you mention a problem with multiple wide matches:

```
    # Following config will match widely causing any matches that follows invalid. Hence commented out.
```

Isn't the issue the same here?

Reply:
I considered it as if the forwarder is not forwarding it would just print all out for debugging purposes. And if the users had something to do with the outputs won't it be done using the aggregator. 
What do you think is better.

Reply:
Another solution would be to move

        {{ .Values.aggregator.extraConfig | indent 4 }}

it up to line 83. Then default match all won't affect user configuration. 
If this is considered then same can be done in the aggregator as well.